Makefile

```diff
@@ -55,6 +57,11 @@ LLVM_CXX_FLAGS += -DLLVM_VERSION=$(LLVM_VERSION_TIMES_10)
 WITH_NATIVE_CLIENT ?= $(findstring nacltransforms, $(LLVM_COMPONENTS))
 WITH_X86 ?= $(findstring x86, $(LLVM_COMPONENTS))
 WITH_ARM ?= $(findstring arm, $(LLVM_COMPONENTS))
+ifeq ($(LLVM_VERSION_TIMES_10),39)
```

Comment:
This check appears to be working for all but one Travis CI config: https://travis-ci.org/halide/Halide/builds/136474809

Interestingly... it works for one LLVM_VERSION=3.8.0, but not another!?